code/game/objects/items/stacks/stack.dm

```diff
@@ -27,6 +27,8 @@
 	var/merge_type = null // This path and its children should merge with this stack, defaults to src.type
 	var/recipe_width = 400 //Width of the recipe popup
 	var/recipe_height = 400 //Height of the recipe popup
+	/// What sort of table is made when applying this stack to a frame?
+	var/table_type = null
```

Comment:
```suggestion
	var/table_type
```

Reply:
I wanted to explicitly state it here for clarity but I'm happy to remove it if needed

Reply:
since this var shouldn't be null in-game when we actually use stacks, I'd go ahead and remove the manual null definition here

Reply:
It should be null for a number of stacks, only a select few stacks have this var as not-null

Reply:
then leave it manually defined as null I guess